sunbeam-python/sunbeam/commands/validate.py

```diff
@@ -0,0 +1,47 @@
+# Copyright (c) 2022 Canonical Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+
+import click
+from rich.console import Console
+from snaphelpers import Snap
+
+from sunbeam.clusterd.service import ClusterServiceUnavailableException
+from sunbeam.plugins.validation.plugin import ValidationPlugin
+
+LOG = logging.getLogger(__name__)
+console = Console()
+snap = Snap()
+
+
+@click.command()
+def validate() -> None:
+    """Run tempest's validate action."""
+    validation_plugin = ValidationPlugin()
+    try:
+        enabled = validation_plugin.enabled
+    except ClusterServiceUnavailableException:
+        LOG.debug(
+            "Failed to query for plugin status, is cloud bootstrapped ?",
+            exc_info=True,
+        )
+        enabled = False
+
+    if not enabled:
+        console.print(f"'{validation_plugin.name}' plugin is not enabled.")
+        return
+
+    console.print("`sunbeam validate` is not implemented.")
```

Comment:
not-blocking:
```suggestion
def validate() -> None:
    """Run tempest's validate action."""
    validation_plugin = ValidationPlugin()
    try:
        enabled = validation_plugin.enabled
        console.print("`sunbeam validate` is not implemented.")
    except ClusterServiceUnavailableException:
        LOG.debug(
            "Failed to query for plugin status, is cloud bootstrapped ?",
            exc_info=True,
        )
        console.print(f"'{validation_plugin.name}' plugin is not enabled.")
```

Reply:
I will remove this file altogether because it provides a bad user experience: you can run `sunbeam validate` without the validation plugin.